src/FNA3D_Driver_D3D11.c

```diff
@@ -56,9 +56,17 @@
 #include <dxgi.h>
 #endif
 
+#ifndef DXGI_SWAP_EFFECT_FLIP_DISCARD
+#define DXGI_SWAP_EFFECT_FLIP_DISCARD 4
+#endif
+
+#ifndef DXGI_SWAP_CHAIN_FLAG_ALLOW_TEARING
+#define DXGI_SWAP_CHAIN_FLAG_ALLOW_TEARING 2048
+#endif
+
 #ifndef DXGI_PRESENT_ALLOW_TEARING
 #define DXGI_PRESENT_ALLOW_TEARING 0x00000200UL
-#endif /* DXGI_PRESENT_ALLOW_TEARING */
+#endif
```

Comment:
Turns out these values are enums rather than defs, which is why I ended up doing the magic number thing - as a result these should now be covered and can be removed.

Reply:
Addressed in latest commit.